### PR TITLE
If more than one title is given, assume second one is subtitle.

### PR DESCRIPTION
**Pull request by Lukas0907**

### CODE REVIEW

File: feeds/loaders.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ class BaseItemLoader(ItemLoader):
     id_in = MapCompose(str.strip)
 
     title_in = MapCompose(str.strip)
+    title_out = Join(': ')
```
Comment (nblock) [inline feeds/loaders.py]: This breaks in case of more than two items.

**Comment by Lukas0907** [comment]
Yes, that's true.

We could alternatively use `Join(' - ')`.

**Comment by Lukas0907** [comment]
Rebase on top of #57 and remove `or None`.

File: feeds/spiders/falter_at.py [hunk 2]
```diff
@@ -61,11 +61,7 @@ def parse_archive_search(self, response):
                 il.add_value('author_name', author)
             except IndexError:
                 pass
-            if item['subtitle']:
-                title = '{}: {}'.format(item['title'], item['subtitle'])
-            else:
-                title = item['title']
-            il.add_value('title', title)
+            il.add_value('title', item['title'])
```
Comment (nblock) [inline feeds/spiders/falter_at.py]: Isn't the item['subtitle'] missing here?

Comment (Lukas0907) [inline feeds/spiders/falter_at.py]: It is deliberately missing here since the subtitle is also part of the main content.

**Pull request merged**